Autodesk/Revit/README.md

```diff
@@ -1,39 +1,28 @@
+
 The Rhino Inside® technology allows Rhino and Grasshopper to be embedded within other products.
 
 This folder contains a sample project that demonstrates:
 
-1. Starting Rhino and Grasshopper as a Revit Addon.
+1. Starting Rhino, Grasshopper and the Rhino.Python Editor as a Revit Addon.
 2. Call directly into the native Revit APIs from a Grasshopper or Rhino plugin.
-2. Grasshopper definitions can be opened and previewed in Rhino within the Revit process.
-3. Object can be natively created by Rhino or Grasshopper within Revit through the API.
+2. Grasshopper definitions can be opened within the Revit.
+3. Includes a Revit Toolbar in Grasshopper with Revit specific components
+3. Object can be natively created by Rhino or Grasshopper within Revit.
 
 [**Questions?** *Visit the discussion forum...*](https://discourse.mcneel.com/c/serengeti/inside)
 
 ## Getting started
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
-
-### Prerequisites
-* Git client. ([download](https://git-scm.com/downloads))
-* Visual Studio 2017 (15.9 or above) ([download](https://visualstudio.microsoft.com/downloads/))
-* .NET Framework 4.7 ([download](https://www.microsoft.com/net/download/visual-studio-sdks))
-* Rhino WIP ([download](https://www.rhino3d.com/download/rhino/wip))
-* Autodesk Revit 2017-2019 ([download](https://www.autodesk.com/products/revit/free-trial))
-* Add this link to your bookmarks ;) ([API docs](https://www.apidocs.co/apps/))
-
-### Getting Source & Build
-1. Clone the repository. At the command prompt, enter the following command:
-```
-git clone --recursive https://github.com/mcneel/rhino.inside.git rhino.inside
-```
-2. In Visual Studio: open _rhino.inside\Autodesk\Revit\RhinoInside.Revit.sln_.
-3. Set the Solution Configuration drop-down according the Revit version you have installed. This will properly link the correct Revit API libraries to the project.
-4. Navigate to _Build_ > _Build Solution_ to begin your build.
+Grab the [latest installer](https://ci.appveyor.com/api/projects/mcneel/rhino-inside/artifacts/Autodesk%2FRevit%2Fsrc%2FRhinoInside.Revit.Setup%2Fbin%2Fx64%2FRelease%2FRhinoInside.Revit.msi?branch=master) (requires [Rhino WIP](https://www.rhino3d.com/download/rhino/wip) and Revit 2017-2019)
+
+If you are looking to contribute, customize or fork this source go to the [build from source instructions](build-source.md).
 
 ### Installing & Uninstalling
-The project is configured to copy .addon file as well as output files to the folder _"%APPDATA%\\Autodesk\\Revit\\Addins\\%RevitVer%\\RhinoInside.Revit\\"_ folder in order to make Revit load this addon next time it runs.
+The project is configured to copy a .addon file as well as the plugin files to the folder _"%APPDATA%\\Autodesk\\Revit\\Addins\\%RevitVer%\\RhinoInside.Revit\\"_ folder in order to make Revit load this addon next time it runs.
```

Comment:
```suggestion
The installer copies the necessary files to the _"%APPDATA%\\Autodesk\\Revit\\Addins\\<revit_version>\\"_ folder (for each supported version). Restart Revit to load the add-on.
```

Reply:
I think it's better to describe the installer's behaviour and let build-source.md get into the weeds